--- CMakeLists.txt.orig	2021-01-17 16:23:21.923686793 +1000
+++ CMakeLists.txt	2021-01-17 16:23:58.584764324 +1000
@@ -1,12 +1,12 @@
 cmake_minimum_required(VERSION 2.8)
 project(ParMETIS)
 
-set(GKLIB_PATH METIS/GKlib CACHE PATH "path to GKlib")
-set(METIS_PATH METIS CACHE PATH "path to METIS")
+set(GKLIB_PATH metis/GKlib)
+set(METIS_PATH metis)
 
 # Symlink ./metis to wherever metis is. This allows files to be
 # included from metis/libmetis/.
-execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${METIS_PATH} metis)
+# execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${METIS_PATH} metis)
 
 # Search for MPI.
 # GK commented this out as it seems to be creating problems
@@ -23,7 +23,9 @@
   set(ParMETIS_LIBRARY_TYPE STATIC)
 endif()
 
-include(${GKLIB_PATH}/GKlibSystem.cmake)
+# include(${GKLIB_PATH}/GKlibSystem.cmake)
+
+find_library(METIS_LIB NAMES metis${PLAT} PATHS ${LIBMETIS_DIR})
 
 # List of paths that the compiler will search for header files.
 # i.e., the -I equivalent
@@ -33,10 +35,10 @@
 include_directories(${METIS_PATH}/include)
 
 # List of directories that cmake will look for CMakeLists.txt
-add_subdirectory(${METIS_PATH}/libmetis ${CMAKE_BINARY_DIR}/libmetis)
+# add_subdirectory(${METIS_PATH}/libmetis ${CMAKE_BINARY_DIR}/libmetis)
 add_subdirectory(include)
 add_subdirectory(libparmetis)
-add_subdirectory(programs)
+#add_subdirectory(programs)
 
 # This is for testing during development and is not being distributed
 #add_subdirectory(test)
